config/defaults-config.json

```diff
@@ -108,5 +108,7 @@
   "CLEANUP_TELEGRAM": true,
   "CLEANUP_TIME_TG":"10",
   "CLEANUP_DATABASE": true,
-  "CLEANUP_TIME_DB":"10080"
+  "CLEANUP_TIME_DB":"10080",
+  "INLINESEARCH_LATITUDE":48.8582012,
```

Comment:
Should not have a default value since there is no default that would be reasonable. This is one of the reasons for further down recommending against using coordinates for sorting.